manifests/init.pp

```diff
@@ -123,10 +123,8 @@
   Optional[Array] $code_manager_data_filter              = undef,
 ) {
 
-  $agent_node = $facts['fqdn'] != $facts['puppet_server']
-
   # Account for the differences in Puppet Enterprise and Open Source and Agent
-  if $agent_node {
+  if $facts['splunk_hec_agent_only_node'] {
```

Comment:
For DRY purposes, would it have been better to set `$agent_node` to `$facts['splunk_hec_agent_only_node']` above and kept the conditionals the way they were?

Reply:
Done.